src/components/localization/InstallationLanguage.jsx

```diff
@@ -329,15 +329,17 @@ const InstallationLanguage = ({ idPrefix, setIsFormValid, setStepNotification })
     );
 };
 
-export const usePage = () => {
+const PageTitle = () => {
     const osRelease = useContext(OsReleaseContext);
-    const isBootIso = useContext(SystemTypeContext) === "BOOT_ISO";
-
-    return ({
-        component: InstallationLanguage,
-        id: "installation-language",
-        isHidden: !isBootIso,
-        label: _("Welcome"),
-        title: cockpit.format(_("Welcome to $0"), osRelease.NAME),
-    });
+    return cockpit.format(_("Welcome to $0"), osRelease.NAME);
 };
+
+export class Page {
```

Comment:
The "convert each page definition to a class" commit could use some justification, i.e. the commit message could say "why?". That's not obvious.

Reply:
It's an architectural decision that I arbitrarily met to improve readability. I am open to discuss this. Generally I want that each Wizard page has an standardized exported 'something', ie class, object whatever, so that the Wizard can be agnostic of the pages is contains. 

This opens the road for supporting in the future add-ons.